src/kbmod/search/kernels.cu

```diff
@@ -23,6 +23,8 @@
 
 namespace search {
 
+__host__ __device__ bool device_pixel_valid(float value) { return ((value != NO_DATA) && !isnan(value)); }
```

Comment:
I'm just a little bit confused why we're keeping the NO_DATA around. If we set NO_DATA to nan and check isnan does that break things. This can be a matter for a different PR though.

Reply:
The main reason is that removing NO_DATA as a flag the code sets will be more extensive. I will work on that in a later PR. But I wanted to get the code to support checking for NaNs in the short term.